Source/WTF/wtf/AutodrainedPool.h

```diff
@@ -28,9 +28,7 @@
 
 #pragma once
 
-#if defined(__OBJC__) && !defined(__clang_tapi__)
-#error Please use @autoreleasepool instead of AutodrainedPool.
```

Comment:
Consider replacing this with `#pragma message ...`? So that it at least shows up as a diagnostic somewhere?

Reply:
it would show up erroneously all the time because the header is part of the WTF module; and it doesn’t offer much benefit either since if someone tries to use it outside of Objective C, they just won’t be able too anyways so it’s not like they can mistakenly do something